diagrams-pandoc.cabal

```diff
@@ -37,7 +37,8 @@ library
                        diagrams-svg >= 1.4 && < 1.5,
                        diagrams-core >= 1.4 && < 1.6,
                        hashable >= 1.2 && < 1.5,
-                       svg-builder >= 0.1 && < 0.2
+                       svg-builder >= 0.1 && < 0.2,
+                       pandoc >= 3.0 && < 3.2
```

Comment:
`pandoc` is now a dependency since we want to be able to use markdown for captions, rather than plain text. `pandoc` is used to render such captions with rich text.